content/applications/finance/fiscal_localizations/germany.rst

```diff
@@ -183,13 +49,13 @@ January 2020 to specify some of the content and due to the development of digita
 hosting, paperless companies, etc.).
 
 .. important::
-   Odoo gives you **the means to be compliant with GoBD**.
+   Odoo is certified **GoBD-compliant**.
 
 What do you need to know about GoBD when relying on accounting software?
 ------------------------------------------------------------------------
 
 .. note::
-   If you can, the best way to understand GoBD is to Read the `Official GoBD text
+   If you can, the best way to understand GoBD is to read the `official GoBD text
    <https://ao.bundesfinanzministerium.de/ao/2021/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/anhang-64.html>`_.
    It is a bit long but quite readable for non-experts. But in short, here is what to expect:
```

Comment:
I would remove this note entirely since I added a seealso at the beginning of the section and I find the second sentence a bit weird for documentation.